app/scripts/metamask-controller.js

```diff
@@ -261,9 +261,9 @@ export default class MetamaskController extends EventEmitter {
       }),
       showApprovalRequest: opts.showUserConfirmation,
       typesExcludedFromRateLimiting: [
-        MESSAGE_TYPE.ETH_SIGN,
-        MESSAGE_TYPE.PERSONAL_SIGN,
-        MESSAGE_TYPE.ETH_SIGN_TYPED_DATA,
+        ApprovalType.EthSign,
+        ApprovalType.PersonalSign,
+        ApprovalType.EthSignTypedData,
```

Comment:
We can use this opportunity to also include the types for transaction, watch asset, encryption public key, and decrypt. As they are all new so we don't want changes in behaviour.